invenio_app_rdm/config.py

```diff
@@ -1078,6 +1081,15 @@ def github_link_render(record):
 }
 """User requests search configuration (i.e list of user requests)"""
 
+# Invenio-Checks
+# ==============
+RDM_COMMUNITY_SUBMISSION_REQUEST_CLS = checks_requests.CommunitySubmission
+"""Community submission request class taking into account the Checks feature."""
+RDM_COMMUNITY_INCLUSION_REQUEST_CLS = checks_requests.CommunityInclusion
+"""Community inclusion request class taking into account the Checks feature."""
+RDM_RECORDS_SERVICE_COMPONENTS = DefaultRecordsComponents + [ChecksComponent]
+"""Add Checks Component to record service components."""
```

Comment:
Setting this kind of config here is a bit unconventional, the idea being that since we're in Invenio-App-RDM, we can "bring together" all the configs to setup features.

The alternative is to not do this at all, and rely on documentation on how to enable features.